waves/src/hooks/RateService.tsx

```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext } from "react";
+
+export const RateServiceContext = createContext<number>(20000);
```

Comment:
This will likely store the URL we want to use to talk to the rate service and not the actual rate.

You could do that right away and only fake the internal then you don't have to update this interface once we actually use the rate service.

Reply:
Alternatively, you could make the actual `RateService` be the value of the context and you only depend on an interface. Then you could control which rate service is used from the top of the component tree.